fvtr/python/python.exp

```diff
@@ -105,6 +105,10 @@ set env(TMPDIR) "$FULLPATH/tmp[pid]"
 exec mkdir $env(TMPDIR)
 printit "TMPDIR=\"$env(TMPDIR)\""
 
+# Test for the issue #45
+printit "Executing sqlite3 importing test."
+exec $env(AT_DEST)/bin/python3 -c "import sqlite3"
+
```

Comment:
> As we are now explicitly adding it, this test may run. So we don't need to worry about the test execution.

Looks like I made another mistake: test_sqlite is already being run with your patch.

>  Only move the test inside the python3 IF.

Yes!